### DISCUSSION TITLE
Does pmp-library support multiple thread?

### DISCUSSION THREAD

**Discussion by cshan** [pmp-library/pmp-library#196] [Q&A]
such as openmp

**Comment by dsieger** [comment]
I'm sorry, but your questions is a bit vague. What sort of multi-threading support are you looking for?

PMP uses OpenMP for certain linear solvers, but the data structure itself does not provide any multi-threading facilities, e.g., to modify the mesh data structure in parallel.

**Comment by cshan** [comment]
I need fill holes from cracked mesh, looks like this process is very slow, so I hope it can be in multiple threads.

**Reply by meetrainier** [reply]
I assume your mesh is triangular. Can you provide data ? Hoe many triangles and how long does it take? You are using API or interface?
